src/Microsoft.Azure.WebJobs.ServiceBus/Config/ServiceBusExtensionConfig.cs

```diff
@@ -54,6 +57,20 @@ public void Initialize(ExtensionConfigContext context)
                 throw new ArgumentNullException("context");
             }
 
+            // Register an exception handler for background exceptions
+            // coming from MessageReceivers.
+            Config.ExceptionHandler = (e) =>
```

Comment:
Our setting this here will override the logic used by our default MessageReceiverOptions that we set up. If the user sets ServiceBusConfiguration.MessageOptions to their own instance with their own handler, they completely override, and our handler doesn't get used (by design). For WebJobs scenarios that's what we want - user has full control. For Functions, they can't override our MessageReceiverOptions instance, so our logging will always be in effect.